asf_search/export/jsonlite.py

```diff
@@ -70,25 +52,51 @@ def __init__(self, results):
 
         # need to make sure we actually have results so we can intelligently set __len__, otherwise
         # iterencode behaves strangely and will output invalid json
-        self.first_result = None
-        self.len = 0
-        for p in self.results:
-            if p is not None:
-                self.first_result = p
-                self.len = 1
-                break
+        self.len = 1
 
     def __iter__(self):
         return self.streamDicts()
 
     def __len__(self):
         return self.len
 
+    def get_additional_output_fields(self, product: ASFProduct):
+        umm = product.umm
+        
+        additional_fields = {}
+        for key, path in extra_jsonlite_fields:
+            additional_fields[key] = get_additional_fields(umm, *path)
+
+        if product.properties['platform'].upper() in ['ALOS', 'RADARSAT-1', 'JERS-1', 'ERS-1', 'ERS-2']:
+            insarGrouping = get_additional_fields(umm, *['AdditionalAttributes', ('Name', 'INSAR_STACK_ID'), 'Values', 0])
+            insarStackSize = get_additional_fields(umm, *['AdditionalAttributes', ('Name', 'INSAR_STACK_SIZE'), 'Values', 0])
+            
+            if insarGrouping not in [None, 0, '0', 'NA', 'NULL']:
+                additional_fields['canInsar'] = True
+                additional_fields['insarStackSize'] = insarStackSize
```

Comment:
Is this the only place where insarStackSize is used? What if we moved the `insarStackSize = get_additional_fields ...` line inside the if statement? It feels weird looking it up, and only using it half the time